### PR TITLE
fix(docs): prevent caching when there's a bad fetch (VIV-1195)

### PR DESCRIPTION
**Pull request by YonatanKra**

### CODE REVIEW

**Review by rinaok — CHANGES_REQUESTED** [review]
Screenshot 2023-07-09 at 19 06 52


There is a problem loading fonts and pictures

**Comment by YonatanKra** [comment]
> Screenshot 2023-07-09 at 19 06 52
> There is a problem loading fonts and pictures

Thanks. Strangely it works locally. Looking into it. Thanks again :)

**Comment by YonatanKra** [comment]
@rinaok - can you recheck? Looks good on my end. Then again, it looked good before so... ;)

**Comment by rinaok** [comment]
> icon.resources.vonage.com

@YonatanKra Yes, now it is loading :)

File: apps/docs/assets/bundled-scripts/sw.js [hunk 1]
```diff
@@ -2,14 +2,19 @@ const VERSION = 'SW_VERSION';
 
 console.log(`Vivid Upgraded to Version ${VERSION}!`);
 
+const iconsUrl = 'icon.resources.vonage.com';
```
Comment (rinaok) [inline apps/docs/assets/bundled-scripts/sw.js:5]: Is it possible to make this a global url? It appears in multiple places in the code.

Comment (YonatanKra) [reply apps/docs/assets/bundled-scripts/sw.js:5]: It seems like a bigger issue due to how we our build works. I'll sort it out in a different [issue](https://jira.vonage.com/browse/VIV-1198).

Comment (rinaok) [reply apps/docs/assets/bundled-scripts/sw.js:5]: You can add it to this PR: https://jira.vonage.com/browse/VIV-1128

**Review by rinaok — APPROVED** [review]

**Pull request merged**